pkg/vulnsrc/amazon/amazon.go

```diff
@@ -114,11 +114,15 @@ func (vs VulnSrc) commitFunc(tx *bolt.Tx) error {
 					references = append(references, ref.Href)
 				}
 
+				severity := severityFromPriority(alas.Severity)
 				vuln := types.VulnerabilityDetail{
-					Severity:    severityFromPriority(alas.Severity),
+					Severity:    severity,
 					References:  references,
 					Description: alas.Description,
 					Title:       "",
+					VendorSeverity: types.VendorSeverity{
+						vulnerability.Amazon: severity,
```

Comment:
It always stores the same value as `Severity`.

Reply:
Yes, is that not right?

Reply:
My idea is to use the vulnerability that comes from vendor like this https://github.com/aquasecurity/trivy/compare/vendor-score-priority?expand=1#diff-d85a008c0dbe874b6d2b0caf6eee787cR53-R58 

If this doesn't make sense, please let me know.